### ISSUE TITLE
Authenticated user is not allowed access - GoDaddy changed the API requirements

### ISSUE THREAD

**Issue by rnlgreen** [navilg/godaddy-ddns#32]
Not an issue with godaddy-ddns as such, but in case anyone else tries using the API service for the first time like me, it seems GoDaddy changed the requirements of usage a couple of days ago so that you have to have 50 domains registered with them to be able to use the API.

You'll see this error in the godaddy-ddns logs:
`Authenticated user is not allowed access, StatusCode: 403, StatusCode: 1`

**Comment by navilg** [comment]
@rnlgreen Thanks for reporting. I am not able to find any document about change in API policy. Could you please help me with document or announcement about this change

**Comment by fals3illusion** [comment]
is there any way to get this working?  I was just about to create a thread that it no longer updates ip.
i get the following error in my logs like you mentioned.

"Failed to update record. Error: addRecord Error getting DNS record Error: Authenticated user is not allowed access, StatusCode: 403, StatusCode: 1"

**Comment by navilg** [comment]
I will try to look into some alternatives. Currently it looks like GoDaddy has changed its developer policy and does not allow API access for users having less than 50 domains (I am yet to find a document/announcement/article to verify this).

**Comment by fals3illusion** [comment]
Damn, bummer.  Ill keep checking this post.  Thanks!

**Comment by rnlgreen** [comment]
I don't think there's any documentation or announcement, I just found a thread on Reddit:
[https://www.reddit.com/r/godaddy/comments/1chs1j8/godaddy_access_denied_via_apicall/](url)

There' some confusion around whether the requirement is 50 or 10 domains, and there's also some subscription plan you can get:
[https://www.reddit.com/r/godaddy/comments/1bl0f5r/am_i_the_only_one_who_cant_use_the_api/](url)

**Comment by svenerbe** [comment]
maybe its possible to use OTE environment which is hosted at https://api.ote-godaddy.com/. its possible to add a parameter to specify the ote env?

**Comment by navilg** [comment]
Will check this. Thanks

**Comment by saditude** [comment]
Are there any advancements in finding out a solution?

**Comment by navilg** [comment]
I will need some help from community. I checked on internet but I do not find any alternative and most articles shows old API key ways. I also tried to reach out to GoDaddy customer support, but looks like since I have moved my domain out of GoDaddy, they are also not helpful.

Can you please reach out to GoDaddy support team and ask what alternatives they suggest to setup DDNS, if API keys are not working.

**Comment by svenerbe** [comment]
My solution was to move out my domain from godaddy. I reach out the Godaddy api support team. I get this answer:-(

Hi,

We have recently updated the account requirements to access parts of our production Domains API. As part of this update, access to these APIs are now limited:

Availability API: Limited to accounts with 50 or more domains
Management and DNS APIs: Limited to accounts with 10 or more domains and/or an active Premium Discount Domain Club plan.
If you have lost access to these APIs, but feel you meet these requirements, please reply back with your account number and we will review your account and whitelist you if we have denied you access in error.

Please note that this does not affect your access to any of our OTE APIs.

If you have any further questions or need assistance with other API questions, please reach out.

Regards,

API Support Team

**Comment by fals3illusion** [comment]
Yeah, I just moved all my stuff to cloudflare. They have a dns update that works and had to renew my domain name too so I just went with them for everything. Sucks I was with GoDaddy for a long time always liked them. Too bad.

**Comment by Stepyon** [comment]
I would with to see this thread before I registered a domain with GoDaddy, now I have the same issue.

**Comment by davispuh** [comment]
It's such a bullshit move on their part but I didn't know this before bought domain and no idea where else to buy domain I wanted so stuck with them. Technically it's possible to update it thru website way but I would start just telling everyone to avoid them like plague and look into moving to other providers.

**Comment by navilg** [comment]
I have switched to namecheap and https://github.com/navilg/namecheap-ddns-docker works like a charm

**Comment by codemonstur** [comment]
I'm affected by this new GoDaddy policy. Does anyone know an alternative registrar that has an API that supports querying the DNS records? I need to list, remove and add TXT records. With the namecheap registrar I can't figure out if their API supports setting DNS records because their documentation is so poor.

**Comment by davispuh** [comment]
@codemonstur  https://porkbun.com/ their API works fine and libraries are widely available.

**Comment by codemonstur** [comment]
@davispuh hey thanks that looks nice. But I ultimately chose to go with https://zone.eu.

**Comment by aboliDTskill** [comment]
> support tea

How to contact GoDaddy support Team?

In my case not able to chat , since there is no option for chat bot or chat box. 

![Image](https://github.com/user-attachments/assets/0fe851a8-2b79-4d69-9aef-ba36f0843259)

**Comment by ckrautwald** [comment]
I wonder just how many people GoDaddy has lost due to this. 

I found this post mostly because I started playing with a minirack and following directions for acquiring SSL Certs through NPM and well...I never really liked GoDaddy, but this is some elitist/hoity toity BS man.

**Comment by AnastaZIuk** [comment]
wow this is hilarious, time to change provider

**Comment by ckrautwald** [comment]
> wow this is hilarious, time to change provider

Might I suggest Porkbun? That's who I switched to. It works great with my NPM implementation.

**Comment by AnastaZIuk** [comment]
> > wow this is hilarious, time to change provider
> 
> Might I suggest Porkbun? That's who I switched to. It works great with my NPM implementation. 

I will give it a look, thanks!
